libcontainer/dmz/Makefile

```diff
@@ -1,7 +1,19 @@
-# Get CC values for cross-compilation.
+# Get GO, GOARCH and CC values for cross-compilation.
 include ../../cc_platform.mk
 
-# We use the flags suggested in nolibc/nolibc.h, it makes the binary very small.
+# List of GOARCH that nolibc supports, from:
+# 	https://go.dev/doc/install/source#environment (with GOOS=linux)
+#
+# See nolibc supported arches in ./nolibc/arch-*.h
+NOLIBC_GOARCHES := 386 amd64 arm arm64 loong64 ppc64le riscv64 s390x
+
+ifeq ($(GOARCH), $(findstring $(GOARCH), $(NOLIBC_GOARCHES)))
```

Comment:
This looks for a substring and won't work in all cases (e.g. it will match `arm` if the list will contain `arm64` entry which is not correct).

The proper way is to use filter, which treats its last argument  as a space-separated list. See the top-level Makefile for examples.

Reply:
Ohh, great catch, thanks! Fixed it and used the style from the top-level Makefile now :)